### DISCUSSION TITLE
[Feature] Add support to link build assets

### DISCUSSION THREAD

**Discussion by bestlinuxgamers** [woodpecker-ci/woodpecker#2298] [Request a Feature]
### Clear and concise description of the problem

Currently it is not possible for Woodpecker plugins to integrate with the Ui. So it is for example not possible to download builds (which were saved with the [drone s3 plugin](http://plugins.drone.io/drone-plugins/drone-s3/), for example) directly from the Ui.

### Suggested solution

A simple solution would be to give plugins the ability to store links in a build. These links are then displayed in the Ui. So, as a solution for my example, a download link to the build could be stored.

### Alternative

A long term solution would be to allow plugins to integrate with the Ui. But since this is too much effort for now, it should be discussed in a separate feature request.

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.github.io/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by anbraten** [comment]
should probably be solved by #1006
